### ISSUE TITLE
In vagrant long_descriptions, example code bash variables get substituted

### ISSUE THREAD

**Issue by deargle** [ruzickap/packer-templates#153]
This is correct most of the time, but breaks the script showing how to pull the latest version of vagrant. 

Specifically, 
```
VAGRANT_LATEST_VERSION=$(curl -s https://checkpoint-api.hashicorp.com/v1/check/vagrant\
  | jq -r -M '.current_version')
dnf install https://releases.hashicorp.com/vagrant/${VAGRANT_LATEST_VERSION}/vagrant_${VAGRANT_LATEST_VERSION}_x86_64.rpm
```

gets rendered on vagrantup.com as:

![image](https://user-images.githubusercontent.com/1174653/89105532-229b6b00-d3df-11ea-9012-6c370c02f9e5.png)

I'm not sure how to fix it, but relevant lines:

https://github.com/ruzickap/packer-templates/blob/19143b1b07079cd8874194c75bce97415a48409d/upload_box_to_vagrantcloud.sh#L80

https://unix.stackexchange.com/a/294400

**Comment by ruzickap** [comment]
Thanks for reporting this - great catch.
I'll fix it asap...
I found some solutions here: https://stackoverflow.com/questions/24963705/is-there-an-escape-character-for-envsubst

**Comment by ruzickap** [comment]
The issue should be fixed by #155

**Comment by ruzickap** [comment]
It should be fixed in all descriptions.
Thanks for reporting it.
